docs/how_to/convert_video_formats_with_ffmpeg.rst

```diff
@@ -0,0 +1,278 @@
+.. _convert_video_formats_with_ffmpeg:
+
+Converting Video Formats with FFmpeg for DANDI Upload
+=====================================================
+
+This guide explains how to convert bespoke video file formats to DANDI-compatible formats using FFmpeg,
+enabling successful conversion to NWB format and upload to the DANDI Archive.
+
+Why Convert Video Formats?
+--------------------------
+
+DANDI requires videos to be in specific formats for upload. The supported video file extensions are:
+
+- ``.mp4`` (recommended)
+- ``.avi``
+- ``.wmv``
+- ``.mov``
+- ``.flv``
+- ``.mkv``
+
+.. note::
+   NeuroConv's video interface currently lists ``.flx`` instead of ``.flv`` in its supported formats.
+   If you have ``.flv`` files, they should work fine with DANDI after conversion, but you may need
+   to rename them to ``.flx`` for NeuroConv or convert them to ``.mp4`` using the examples below.
```

Comment:
I believe the .flx extension listed on the video interfaces is actually a typo. It should be.flv. Please adjust the interface code accordingly.